tests/bluetooth/df/connection_cte_tx_params/prj.conf

```diff
@@ -0,0 +1,31 @@
+# CONFIG_TEST=y
+# CONFIG_ZTEST=y
+
+# CONFIG_BT=y
+# # It does not matter for the test whether central or peripheral is used.
+# # Each of these roles selects BT_CONN.
+# CONFIG_BT_CENTRAL=y
+
+# CONFIG_BT_CTLR=y
+# CONFIG_BT_LL_SW_SPLIT=y
+
+# CONFIG_BT_CTLR_DF=y
+
+CONFIG_TEST=y
+CONFIG_ZTEST=y
+
+CONFIG_BT=y
+
+CONFIG_BT_CENTRAL=y
+# CONFIG_BT_SMP=y
+# CONFIG_BT_GATT_CLIENT=y
```

Comment:
Should these and the others that are commented out from this file be removed?

Reply:
Yes they should. I've forgotten to clean it up after debugging.